examples/proportion_test.py

```diff
@@ -16,76 +16,81 @@
+
     made_purchase = np.concatenate([made_purchase_got_called, made_purchase_no_calls])
-    
-    is_female = stats.bernoulli.rvs(0.5, size=100000)
-    got_called = np.concatenate([np.ones(51342), np.zeros(48658)])
-    design_matrix = np.hstack([np.ones((100000, 1)),
+
+    is_female = dist.Bernoulli(0.5).sample(key3, sample_shape=(num_calls + num_no_calls,))
+    got_called = np.concatenate([np.ones(num_calls), np.zeros(num_no_calls)])
+    design_matrix = np.hstack([np.ones((num_no_calls + num_calls, 1)),
                                got_called.reshape(-1, 1),
                                is_female.reshape(-1, 1)])
-    
+
     return design_matrix, made_purchase
 
 
 def model(design_matrix: np.ndarray, outcome: np.ndarray = None) -> None:
     """
     Model definition: Log odds of making a purchase is a linear combination
-    of covariates. Specify aNormal prior over regression coefficients.
+    of covariates. Specify a Normal prior over regression coefficients.
     :param design_matrix: Covariates. All categorical variables have been one-hot
         encoded.
     :param outcome: Binary response variable. In this case, whether or not the
         customer made a purchase.
     """
-    
-    # Use multivariate normal prior in case we want to add more covariates later.
+
     beta = numpyro.sample('coefficients', dist.MultivariateNormal(loc=0.,
                                                                   covariance_matrix=np.eye(design_matrix.shape[1])))
     logits = design_matrix.dot(beta)
-    
+
     with numpyro.plate('data', design_matrix.shape[0]):
-            numpyro.sample('obs', dist.Bernoulli(logits=logits), obs=outcome)
-    
+        numpyro.sample('obs', dist.Bernoulli(logits=logits), obs=outcome)
+
 
 def print_results(coef: np.ndarray, interval_size: float = 0.95) -> None:
     """
     Print the confidence interval for the effect size with interval_size
     probability mass.
     """
 
-    baseline = hpdi(coef[:, 0], prob=0.95)
-    effect_size = hpdi(coef[:, 1], prob=0.95)
-    
-    baseline_response = expit(baseline)
-    response_with_calls = expit(baseline + effect_size)
-    
-    impact_on_probability = response_with_calls - baseline_response
-    
+    baseline_response = expit(coef[:, 0])
+    response_with_calls = expit(coef[:, 0] + coef[:, 1])
+
+    impact_on_probability = hpdi(response_with_calls - baseline_response, prob=0.95)
+
+    effect_of_gender = hpdi(coef[:, 2], prob=0.95)
+
     print(f"There is a {interval_size * 100}% probability that calling customers "
           "increases the chance they'll make a purchase by "
           f"{(100 * impact_on_probability[0]):.2} to {(100 * impact_on_probability[1]):.2} percentage points."
-         )
-    
+          )
+
+    print("There is a 95% probability the effect of gender on the log odds of conversion "
+          f"lies in the interval ({effect_of_gender[0]:.2}, {effect_of_gender[1]:.2f})."
```

Comment:
Oh, I just learn that we can format f-string like this. Nice!

Reply:
Just be careful to avoid them in Pyro since Pyro still supports Python 3.5 which doesn't allow f-strings.